### PR TITLE
Features works section

### PR DESCRIPTION
**Pull request by kensteph**
For this Pull Request I 
-Add the Works Section and use ul, li tags  for the categories and style the elements
-Add the About myself Section and style the elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,6 +65,196 @@ <h1 class="grettings">I’m Kender Glad to see you!</h1>
...
+
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!-- ABOUT SECTION -->
+    <section class="about-section">
+      <div class="about-content">
+        <h1 class="about-title">About Myself</h1>
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest you only put 1 H1 header on your web page and then follow the sequence H2, H3, etc. Depending on the importance, this action will greatly help the accessibility of your website.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @kensteph , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost5

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kensteph) [reply index.html]: Thank you for your review. I will remove it and replace it by an H2 tag.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @kensteph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**